tests/drivers/rtc/rtc_api/src/test_y2k.c

```diff
@@ -10,6 +10,10 @@
 
 #include <time.h>
 
+#ifdef CONFIG_SOC_FAMILY_STM32
+#error STM32 RTC does not support years before 2000
+#endif  /* CONFIG_SOC_FAMILY_STM32 */
+
```

Comment:
This really is not required, the file is already excluded in the CMakeLists.txt, with a comment stating why